### ISSUE TITLE
Turn off translation punctuation entirely

### ISSUE THREAD

**Issue by Funktionar** [DeepLcom/deepl-python#43]
I really want to turn off punctuation completely on German language and others that I use. Using any of the preserve_formatting="1" or split_sentences='off' won't work.

**Comment by daniel-jones-deepl** [comment]
Hi @Funktionar, thanks for creating the issue. I'd like to understand your problem better, could you please give an example sentence that shows this problem?

**Comment by FOehlschlaeger** [comment]
Hello there, I would assume that @Funktionar means something like the following example. 
Sentence: "This is a house, which is red. The boy said: 'This is my house.'"

Without punctuation for translation: "This is a house which is red The boy said This is my house", so completely removing all punctuation of the strings to be translated.

**Comment by Funktionar** [comment]
Yes that's precisely what I'm talking about @FOehlschlaeger, especially with german language, example: 
"Die Vögel sind um diese Jahreszeit sind ganz nett, aber ich weiß nicht, wann sie abfliegen" -> "Die Vögel sind um diese Jahreszeit sind ganz nett aber ich weiß nicht wann sie abfliegen" I want the text to be with completely free of punctuation and it's not possible via the API.
